plugins/lookup/passwordstore.py

```diff
@@ -331,7 +331,6 @@ def check_pass(self):
         try:
             self.passoutput = to_text(
                 check_output2([self.pass_cmd, 'show'] +
-                              (['--password'] if self.backend == 'gopass' else []) +
```

Comment:
This was intentionally added for gopass in #4780. CC @grembo - can you explain why this was added?